ormconfig.json

```diff
@@ -0,0 +1,15 @@
+[
+  {
+    "type": "postgres",
+    "host": "localhost",
+    "port": 5432,
+    "username": "postgres",
+    "password": "postgres",
+    "database": "timesheetsnew",
```

Comment:
How does this relate to `.env` file? Are these repeated?
Also are we still on `timesheetsnew`

The other thing I saw with the ansible deployment is that the username and password defaults are `timesheets_db_user` and `timesheets_db_pass`
https://gerrit.opencord.org/plugins/gitiles/ansible/role/timesheets/+/refs/heads/master/defaults/main.yml

Reply:
N cant access migrations from the app.module. Had to move to a json file to get it to work.
I will be removing the database name and all that soon, as we have a timesheets email, and these should be put in after the pull.

I don't know anything about the ansible db username and pass. I would think these would change when we have postgreSQL server setup, or maybe this is what Zack chose.